tests/app/templating/test_summary_context.py

```diff
@@ -91,10 +97,11 @@ def test_build_view_context_for_summary(self):
         variables = None
 
         context = build_view_context_for_final_summary(self.metadata, self.schema, self.answer_store,
-                                                       self.schema_context, self.block_type, variables, csrf_token)
+                                                       self.schema_context, self.block_type, variables, csrf_token,
+                                                       self.rendered_block)
 
         self.check_context(context)
-        self.assertEqual(len(context['summary']), 4)
+        self.assertEqual(len(context['summary']), 5)
         self.assertTrue('is_view_submission_response_enabled' in context['summary'])
```

Comment:
We should also add a test to ensure new context is here.
`self.assertTrue('collapsible' in context['summary'])`